docs/config.rst

```diff
@@ -41,3 +41,20 @@ An environment variable like ``REACT_APP_FOO`` controls the setting ``FOO``.
     :default: ``https://${OIDC_DOMAIN}/userinfo``
 
    The audience for the access token that is generated by Auth0.
+
+
+.. envvar:: REACT_APP_REFRESH_AUTH_SESSION_SECONDS
+
+    :default: 300
+
+    How often to issue a silent authentication refresh. Technically, when you're logged in,
+    an infinite loop is run that refreshes your access token forever. And this number is
+    number of seconds to sleep between each check.
+
+.. envvar:: REACT_APP_REFRESH_PREEMPTIVE_SECONDS
```

Comment:
This variable name doesn't mention auth or sessions. Perhaps `REACT_APP_AUTH_REFRESH_PREEMPTIVE_SECONDS`? This is all getting really long though. Mostly I think it should mention auth in some way.

PS: I really like that y'all are putting units in the setting names. :+1:

Reply:
Good point @mythmon about the need for mentioning "AUTH" in this one.

I'd rather it be long  than unclear.

@magopian Can you second mythmon's point and I'll make a final pass on this.